nailgun/entities.py

```diff
@@ -2210,6 +2212,9 @@ def path(self, which=None):
     def add_puppetclass(self, synchronous=True, **kwargs):
         """Add a Puppet class to host
 
+        Here is an example of how to use this method::
+            hostgroup.add_puppetclass(data={'puppetclass_id': puppet.id})
+
```

Comment:
I think a benefit of using `**kwargs` is that katello can add/change/remove expected parameters, and the signature of this method doesn't need to be updated to reflect the change in katello's expectations. (Mind you, there's already plenty of tight coupling between nailgun and katello's API. But there's no need to add more coupling than necessary, right?)